Example/EarlGreySnapshots_Tests/SnapshotControllerFactoryTests.swift

```diff
@@ -0,0 +1,61 @@
+@testable import EarlGreySnapshots
+import XCTest
+
+class SnapshotControllerFactoryTests: XCTestCase {
+
+    var sut: SnapshotControllerFactory!
+    var info: SnapshotControllerInfo!
+
+    override func setUp() {
+        super.setUp()
+
+        sut = SnapshotControllerFactory()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+
+        sut = nil
+        info = nil
+    }
+
+    func testControllerHasCorrectUsesDrawViewHierarchyInRectSettingSet() {
+        info = SnapshotControllerInfo(testName: "TestName",
+                                      deviceAgnostic: false,
+                                      recordMode: false,
+                                      imagesDirectory: "")
+
+        let controller = sut.makeSnapshotController(withInfo: info)
+
+        XCTAssertFalse(controller.usesDrawInRect)
+    }
+
+    func testControllerHasCorrectDeviceAgnosticSettingSet() {
+        info = SnapshotControllerInfo(testName: "", deviceAgnostic: false, recordMode: false, imagesDirectory: "")
+        let firstController = sut.makeSnapshotController(withInfo: info)
+        info = SnapshotControllerInfo(testName: "", deviceAgnostic: true, recordMode: false, imagesDirectory: "")
+        let secondController = sut.makeSnapshotController(withInfo: info)
```

Comment:
You should either use different global `info's` or even better use them as local variables :wink: